plugins/lua/sort/places.lua

```diff
@@ -33,9 +34,15 @@ local language_name_types = {
     [df.language_name_type.SymbolFood] = 'Inn Tavern',
     [df.language_name_type.Temple] = 'Temple',
     [df.language_name_type.Hospital] = 'Hospital',
-    [df.language_name_type.Guildhall] = 'Guildhall'
+    [df.language_name_type.Guildhall] = 'Guildhall',
+    [df.language_name_type.Library] = 'Library'
```

Comment:
you can end this line with a comma too to reduce the diff footprint of future edits